README.md

```diff
@@ -142,6 +142,16 @@ Similarly, a `QBWC::Worker#handle_response` method cannot access variables that
 
 ## Sessions ##
 
+### Handling errors ###
+
+When any error response is received from QuickBooks Web Connector, by default the session will be halted and no further jobs or their requests requests will be processed. You can change this default behavior to continue processing jobs and their requests by assigning this in `config/initializers/qbwc.rb`:
```

Comment:
- I think this may make more sense as its own section.
- "session will be halted" doesn't really mean much
- "requests requests"
- I don't think you need a code sample for this
- Needs to specify exactly what :continue does.

Suggested new text: "By default, when an error response is received from QuickBooks, no further requests will be processed in the current job or in subsequent jobs. To have qbwc continue with the next request after an error, set `on_error` to `:continue` in `config/initializers/qbwc.rb".